Packs/CommonScripts/Scripts/CommandLineAnalysis/CommandLineAnalysis.py

```diff
@@ -0,0 +1,669 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import base64
+import re
+import ipaddress
+import json
+
+
+def is_base64(s):
```

Comment:
All of the functions arguments in this script needs type annotations.
See example error:
> /Users/sfainberg/dev/demisto/content/Packs/CommonScripts/Scripts/CommandLineAnalysis/CommandLineAnalysis.py:9:0: W9014 is_base64: Function arguments are missing type annotations. Please add type annotations

Also, using none-descriptive command arguments such as "s" is considered bad practice, these names are typically confusing and does not add information to whoever is reading the code.

Suggested function declaration would look something like this:
```python
def is_base64(base64_string: str) -> bool:
```

Please implement this logic for all functions in this script, let me know if you need help.